src/Components/Icons/EditandDelete.jsx

```diff
@@ -94,6 +94,7 @@ const EditandDelete = (props) => {
       },
     });
     //console.log("working");
+    //review
```

Comment:
Common Input Component use

Reply:
okay I will make a component for common input